packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.types.ts

```diff
@@ -0,0 +1,12 @@
+import { ContextualMenuItem } from './ContextualMenuItem';
+import { IContextualMenuItem } from './ContextualMenu.types';
+import { IMenuItemClassNames } from './ContextualMenu.classNames';
+
+export interface IContextualMenuItemProps
+  extends React.HTMLAttributes<IContextualMenuItemProps> {
+  item: IContextualMenuItem;
```

Comment:
Can we add docs here? Even if it's super simple content, stubbing it out will make updating later easier.